langtest/langtest.py

```diff
@@ -164,7 +165,19 @@ def __init__(
                 model = resource_filename("langtest", "data/textcat_imdb")
 
         elif (
-            type(data) is dict
+            isinstance(data, dict)
+            and hub in self.SUPPORTED_HUBS_HF_DATASET_NER
+            and task == "ner"
+        ):
+            self.data = HuggingFaceDataset(data["name"], task=task).load_data(
+                feature_column=data.get("feature_column", "tokens"),
+                target_column=data.get("target_column", "ner_tags"),
+                split=data.get("split", "test"),
+                subset=data.get("subset", None),
+            )
```

Comment:
What we could probably do here is to remove the default parameter values of the `load_data` method and only set them in the `load_data_ner`, `load_data_XXX` methods. That is gonna easier to extend to new tasks.